packages/app/src/components/AgentLoginPromo.module.css

```
@@ -0,0 +1,89 @@
+  border-radius: 4px;
+  cursor: pointer;
+}
+.button:hover:not(:disabled) {
+  background: #f3f4f6;
+}
+.button:focus-visible {
+  outline: 2px solid #2563eb;
+  outline-offset: 1px;
+}
+
+:global([data-color-scheme="dark"]) .icon {
```

Stylelint (17.10.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)